backend/core/entities/user.py

```diff
@@ -103,7 +103,7 @@ class UserSettings(ORMBase, TimestampMixin):
     theme: Mapped[str] = mapped_column(
         String(),
         nullable=False,
-        default=DefaultUserSettings.theme,
+        default=DefaultUserSettings.theme.value,
```

Comment:
Does this work? I believe dataclasses don't need `.value`.

```suggestion
        default=DefaultUserSettings.theme,
```

Reply:
Ahh, that's because theme is `(str, Enum)`, got it. Valeria and I have actually updated the enums to be `(strEnum)` or `(intEnum)`, which don't require `.value`. So u can just change it to strEnum and get rid of .value:

```
class Theme(strEnum):
    LIGHT = "light"
    DARK = "dark"
```